src/amp-story-player/amp-story-player-impl.js

```diff
@@ -302,6 +303,16 @@ export class AmpStoryPlayer {
     this.render_(renderStartingIdx);
   }
 
+  /**
+   * Prerenders the provided story in the player.
+   * @param {string} storyUrl
+   * @public
+   */
+  prerender(storyUrl) {
+    this.autoplay_ = false;
+    this.show(storyUrl).then(() => (this.autoplay_ = true));
```

Comment:
If someone calls `prerender` but have the config `autoplay: false`, shouldn't it keep that behavior?

Reply:
Yep good point, we should probably save the autoplay state to return it to.

> Also, what's the difference between this and just calling show?

It loads a story on the player on prerender mode without playing it. It's basically syntactic sugar for opting out of autoplay and calling show().

An example would be a carousel/grid with multiple stories. The publisher would like to prerender certain story since they know it's the most likely to be clicked. They can just call `player.prerender(storyUrl)` which will start loading the story without playing it, and then when the user actually clicks on something, they'll call `player.play()` to start playing the story.

Reply:
Ok, then I think what you have now is good, since it would start autoplaying when you scroll to it right?